Assets/HotmaxEditorVR/builder/uiClasses/editorModeUI.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
```

Comment:
im not seeing a big reason for this class to exist, if the EditorModeUIController class also exists -- the most important functionality is that it listens to quadrants from the stateManager and sets the correct mode, but that could happen in the EditorModeUIController class and it would make more sense

Reply:
yup, moving it, good idea